src/main/kotlin/sidechain/iroha/consumer/IrohaNetworkImpl.kt

```diff
@@ -35,6 +35,8 @@ class IrohaNetworkImpl(host: String, port: Int) : IrohaNetwork {
         logger.info { "send TX to IROHA" }
 
         // Send transaction to iroha
+        val lst = Endpoint.TxList.newBuilder().addTransactions(protoTx).build()
+        commandStub.listTorii(lst)
         commandStub.torii(protoTx)
```

Comment:
- update iroha_javalib
- add test with multiple commands in one tx
- add test with failing tx in batch, ensure others are committed
- check comments and descriptions are up to date